src/components/Person.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  let maritalStatus = 'I am not married';
+
+  if (isMarried && sex === 'm') {
+    maritalStatus = `My wife's name is ${partnerName}`;
+  }
+
+  if (isMarried && sex === 'f') {
+    maritalStatus = `My husband's name is ${partnerName}`;
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && `I am ${age}`}</p>
```

Comment:
Це створить пусту p, так робити не варто.
```suggestion
      {age && (
        <p className="Person__age">I am {age}</p>
      )}
```